docs/source/user/parallel.rst

```diff
@@ -224,6 +224,41 @@ give an equivalence parallel implementation using :func:`~numba.guvectorize`,
 it would require a pervasive change that rewrites the code to extract kernel
 computation that can be parallelized, which was both tedious and challenging.
 
+Unsupported Operations
+======================
+
+This section contains a non-exhaustive list of commonly encountered but
+currently unsupported features:
+
+#. Concurrent write operations on container types (i.e. lists, sets and
```

Comment:
Should we give it a title for quick look up? e.g. "List operations are not thread safe."

Reply:
Good idea, added similar titles in 57b0028